browser-test/src/support/admin_questions.ts

```diff
@@ -158,7 +158,7 @@ export class AdminQuestions {
     ).toContain('View')
     expect(
       await this.page.innerText(this.selectQuestionTableRow(questionName)),
-    ).toContain('New Version')
+    ).toContain('Edit')
```

Comment:
Can we make this assert a bit more scoped? I believe this will match "Edit draft ->" as well since it's asserting that the contents only includes the text "Edit". We may need to add a bit more scoped of a selector to try and extract out a list of displayed links and then ensure one of them includes the exact text of "Edit".

Reply:
I tried playing with this and am having trouble scoping it down. For instance, this doesn't seem to match:
```
.toEqual('Edit →')
```
Do you have a recommendation for how to best scope it down?

Reply:
> I tried playing with this and am having trouble scoping it down. For instance, this doesn't seem to match:
> 
> ```
> .toEqual('Edit →')
> ```
> 
> Do you have a recommendation for how to best scope it down?

We may have to dive into the DOM a bit more. This may not be better, but we could adjust the selector used for innerText to select that last `td` in the row (https://playwright.dev/docs/selectors#n-th-element-selector). Then we could further subselect all of the `<a>` tags within that row and assert that "Edit" appears.

```
private questionTableRowLinks(questionName: string) {
  return this.selectQuestionTableRow(questionName) + ' td >> nth=-1 > a'
}
```

Then that wouldh ave all "a" tags, which could be used to assert the contents.